source/openapi-admin-v3.yaml

```diff
@@ -5525,7 +5547,71 @@ components:
         error_code:
           type: string
           description: The error type.
-    AtlasCluster:
+    AtlasConfigResponse:
+      type: object
+      properties:
+        config:
+          type: object
+          properties:
+            clusterId:
+              type: number
+              example: 05e821b3975d271289f372e3a
+              description: The service ID.
+            clusterName:
+              type: string
+              example: Cluster0
+              description: |-
+                The service name used to refer to the data source within
```

Comment:
Cluster Name !== Service Name (though eventually they say they will be). This is the actual name of the cluster in Atlas, not e.g. "mongodb-atlas".